From fd2408347c5e71f8ef13c68c8764d307ad71956f Mon Sep 17 00:00:00 2001
From: Raffaele Aquilone <raquiloX>
Date: Mon, 11 Mar 2013 16:21:10 +0100
Subject: [PORT FROM MAIN][BT] Enable lease-change scripts for NAP

BZ: 83501

Enable the HAVE_SCRIPT flag in the build configuration to compile
lease-change scripts, required for DHCP when the device is used as
NAP in the Bluetooth PAN profile.

Change-Id: If1c30d6495b2b772242ca2970e1848627d9bbf26
Orig-Change-Id: I2d766244e8af3585531a206acab7fffdf6ae6f4a
Category: device enablement
Domain: CWS.BT-Common
Origin: internal
Upstream-Candidate: yes
Signed-off-by: Raffaele Aquilone <raquiloX>
Reviewed-on: http://android.intel.com:8080/95991
Reviewed-by: Aquilone, RaffaeleX <raffaelex.aquilone@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Reviewed-by: Gros, Julien <julien.gros@intel.com>
Reviewed-by: Poignet, Cedric <cedric.poignet@intel.com>
Reviewed-by: Bachot, Jean-michel <jean-michel.bachot@intel.com>
Tested-by: Zurmely, PierreX <pierrex.zurmely@intel.com>
---
 src/Android.mk |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/Android.mk b/src/Android.mk
index dfe3f5b..2134b8e 100644
--- a/src/Android.mk
+++ b/src/Android.mk
@@ -10,7 +10,7 @@ LOCAL_MODULE := dnsmasq
 
 LOCAL_C_INCLUDES := external/dnsmasq/src
 
-LOCAL_CFLAGS := -O2 -g -W -Wall -D__ANDROID__ -DNO_IPV6 -DNO_TFTP -DNO_SCRIPT
+LOCAL_CFLAGS := -O2 -g -W -Wall -D__ANDROID__ -DNO_IPV6 -DNO_TFTP -DHAVE_SCRIPT
 LOCAL_SYSTEM_SHARED_LIBRARIES := libc
 LOCAL_SHARED_LIBRARIES := libcutils liblog
 
-- 
1.7.4.1

